addons/l10n_si/data/template/account.tax-si.csv

```diff
@@ -1,193 +1,257 @@
+"","","","","","","","","","","100.00","tax","invoice","+41||+pr_18","gd_acc_160460",""
+"","","","","","","","","","","-100.00","tax","invoice","-25||-ir_20","gd_acc_260460",""
+"","","","","","","","","","","100.00","base","refund","-31a||-ir_25c||-pr_9","",""
+"","","","","","","","","","","100.00","tax","refund","-41||-pr_18","gd_acc_160460",""
+"","","","","","","","","","","-100.00","tax","refund","+25||+ir_20","gd_acc_260460",""
+"gd_taxe_out_0_no_no_1","110","0%","0% exit VAT - other exemptions","0% O Exempt","0.00","percent","sale","tax_group_0","","100.00","base","invoice","+11||+ir_7","","Izstopni DDV 0% - druge oprostitve"
+"","","","","","","","","","","100.00","tax","invoice","","",""
+"","","","","","","","","","","100.00","base","refund","-11||-ir_7","",""
+"","","","","","","","","","","100.00","tax","refund","","",""
+"gd_taxe_out_0_no_no_2","110","0%","0% output VAT - does not go on the books","0% Not in books","0.00","percent","sale","tax_group_0","","100.00","base","invoice","+11||+ir_7","","Izstopni DDV 0% - ne gre v knjigo"
+"","","","","","","","","","","100.00","tax","invoice","","",""
+"","","","","","","","","","","100.00","base","refund","-11||-ir_7","",""
+"","","","","","","","","","","100.00","tax","refund","","",""
+"gd_taxe_out_0_eu_goods_1","130","0%","Exit VAT 0% EU - Goods","0% EU G","0.00","percent","sale","tax_group_0","","100.00","base","invoice","+12||+ir_10a||+rp_a3","","Izstopni DDV 0% EU - blago"
+"","","","","","","","","","","100.00","tax","invoice","","",""
+"","","","","","","","","","","100.00","base","refund","-12||-ir_10a||-rp_a3","",""
+"","","","","","","","","","","100.00","tax","refund","","",""
+"gd_taxe_out_0_eu_goods_2","140","0%","0% EU exit VAT - customs procedure Articles 42 and 43 - goods","0% EU G Art. 42-43","0.00","percent","sale","tax_group_0","","100.00","base","invoice","+12||+ir_10b||+rp_a4","","Izstopni DDV 0% EU - carinski postopek 42. in 43. člen - blago"
+"","","","","","","","","","","100.00","tax","invoice","","",""
+"","","","","","","","","","","100.00","base","refund","-12||-ir_10b||-rp_a4","",""
+"","","","","","","","","","","100.00","tax","refund","","",""
+"gd_taxe_out_0_eu_goods_3","150","0%","0% EU output VAT - triangular supplies - goods","0% EU G Tri","0.00","percent","sale","tax_group_0","","100.00","base","invoice","+12||+ir_11||+rp_a5","","Izstopni DDV 0% EU - tristranske dobave - blago"
+"","","","","","","","","","","100.00","tax","invoice","","",""
+"","","","","","","","","","","100.00","base","refund","-12||-ir_11||-rp_a5","",""
+"","","","","","","","","","","100.00","tax","refund","","",""
+"gd_taxe_out_0_3w_goods_1","170","0%","0% Exit VAT 3. world - goods","0% World","0.00","percent","sale","tax_group_0","","100.00","base","invoice","+11||+ir_7","","Izstopni DDV 0% 3. svet - blago"

```

Comment:
"0% Exit VAT 3. world"? Does this mean "0% Export outside EU?"
I think we should review these English translations to follow our own norms.

Reply:
i talk to christophe about that and he said to put EX (export) instead of world

Reply:
`0% EX G`, yet I don't get what the `3.` stands for.

Reply:
Maybe for third world countries ? but not sure about that, i could just remove it but since it's a description i don't think we really care

Reply:
Also here, I'd say "Export" instead of "Exit" and remove the "3.". since the name "0% EX" doesn't hint at anything about peculiar "third world" incentive taxes.

We as Odoo developers are the primary users of those descriptions - Slovenian businesses will use the translated ones, so I do care, unclear descriptions make it up for confusion later.